aws/data_source_aws_lex_intent.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lexmodelbuildingservice"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func dataSourceAwsLexIntent() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsLexIntentRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Would it make sense to add the remaining fields, such as `conclusion_statement`, `confirmation_prompt`, etc.?

Reply:
So, I looked into this a little. There is a significant amount of duplication since these fields are complex and nested. How can we change them to computed without copying everything?

Reply:
Unfortunately, there isn't a good answer here for avoiding duplication. We can probably merge the PR without that, and create a follow-up issue & PR to add those.